apps/api/src/tasks/sync-cms-cache.task.ts

```diff
@@ -0,0 +1,27 @@
+import { DirectusAdapter } from '@algomart/shared/adapters'
+import { DependencyResolver } from '@algomart/shared/utils'
+import { logger } from '@api/configuration/logger'
+
+export default async function syncCMSCacheTask(registry: DependencyResolver) {
```

Comment:
What's this for exactly? I'd think that anything that gets out of sync would be invalidated immediately. Is that wrong?

Reply:
this is run on the initial load of the application.  Ideally it only runs on application start, rather than hourly

Reply:
Ok yeah, we should probably change that then.

Reply:
@jprincipe ^

Reply:
Ideally if any of them failed the task would fail. Idt any of them need to be run in order, but IMO I'd rather just know the sync was failing for some reason asap and have it prevent the rest because without them all there's going to be problems lurking